web/viewer.js

```diff
@@ -901,6 +902,12 @@ var PDFViewerApplication = {
                      store.get('zoom', self.pdfViewer.currentScale);
           var left = store.get('scrollLeft', '0');
           var top = store.get('scrollTop', '0');
+          rotation = store.get('rotation', '0');
```

Comment:
You should ensure that `rotation` is an integer. Change the line to
`var rotation = store.get('rotation', '0') | 0;` in order to do that.